File: 800/1154A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int a, b, c, d;
    cin >> a >> b >> c >> d;

    int maxi = max({a, b, c, d});

    if (maxi == a)
        cout << (maxi - b) << " " << (maxi - c) << " " << (maxi - d) << "\n";
    else if (maxi == b)
        cout << (maxi - a) << " " << (maxi - c) << " " << (maxi - d) << "\n";
    else if (maxi == c)
        cout << (maxi - b) << " " << (maxi - a) << " " << (maxi - d) << "\n";
    else
        cout << (maxi - b) << " " << (maxi - c) << " " << (maxi - a) << "\n";

    return 0;
}

File: 800/1154A/main.java
import java.util.*;
public class main
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        
        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();
        int d = sc.nextInt();
        
        int maxi = Math.max(a, Math.max(b, Math.max(c,d)));
        
        if(maxi == a)
            System.out.println((maxi-b)+" "+(maxi-c)+" "+(maxi-d));
        else if(maxi == b)
            System.out.println((maxi-a)+" "+(maxi-c)+" "+(maxi-d));
        else if(maxi == c)
            System.out.println((maxi-b)+" "+(maxi-a)+" "+(maxi-d));
        else
            System.out.println((maxi-b)+" "+(maxi-c)+" "+(maxi-a));
    }
}